Cargo.toml

```
@@ -104,10 +104,13 @@ glam = "0.22"
 gltf = "1.1"
 half = "2.3.1"
 image = { version = "0.24", default-features = false }
+infer = "0.15"
```

Maybe we should get into the habit of adding a one-line explanation of new crates, i.e. 

```suggestion
infer = "0.15"  # infer MIME type by checking the magic number signature
```